cpu/stm32/periph/eth.c

```diff
@@ -111,6 +111,53 @@ netdev_t *_netdev;
 static uint8_t _link_state = LINK_STATE_DOWN;
 #endif /* IS_USED(MODULE_STM32_ETH_LINK_UP) */
 
+static void _debug_tx_descriptor_info(void)
+{
+    if (IS_ACTIVE(ENABLE_DEBUG)) {
+        DEBUG("[stm32_eth] TX descriptors:\n");
```

Comment:
I know this is highly unusual (though _you_ were the one putting DEBUG commands in a function :stuck_out_tongue_winking_eye:), but as this is now a function and has the potential to be called from multiple places: make calling line an argument to better track where the TX info was that value?

```suggestion
static void _debug_tx_descriptor_info(int line)
{
    if (IS_ACTIVE(ENABLE_DEBUG)) {
        DEBUG("[stm32_eth:%d] TX descriptors:\n", line);
```

Reply:
I like the idea :-) Done